jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/TCollectorUDPWriter.java

```diff
@@ -68,7 +68,7 @@ public TCollectorUDPWriter(
 			@JsonProperty("tagName") String tagName,
 			@JsonProperty("mergeTypeNamesTags") Boolean mergeTypeNamesTags,
 			@JsonProperty("metricNamingExpression") String metricNamingExpression,
-			@JsonProperty("addHostnameTag") Boolean addHostnameTag,
+			@JsonProperty("addHostnameTag") boolean addHostnameTag,
```

Comment:
The default value here is to be false, so wouldn't that save a forced check later?  This also differs on the default value of true which is expected for openTSDB writes and documentation.